### PR TITLE
[testMigration] Fix type in fs.ts 

### PR DESCRIPTION
**Pull request by ramantehlan**
## Before
DeepinScreenshot_select-area_20200222071637

DeepinScreenshot_select-area_20200222071709

## After
DeepinScreenshot_select-area_20200222071542

### CODE REVIEW

File: packages/fs-observable/__mocks__/fs.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
-const fs = jest.genMockFromModule("fs");
+const fs:any = jest.genMockFromModule("fs");
```
Comment (captainsafia) [inline packages/fs-observable/__mocks__/fs.ts]: Do we need this any type here?

File: packages/fs-observable/__tests__/fs.spec.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ describe("unlinkObservable", () => {
     // File exists so we try to unlink it
     fs.existsSync.mockImplementation(() => true);
     // Unlink fails for some reason (permissions, etc.)
-    fs.unlink.mockImplementation((path, callback) =>
+    fs.unlink.mockImplementation((path: any, callback: (arg0: { message: string; }) => void) =>
```
Comment (captainsafia) [inline packages/fs-observable/__tests__/fs.spec.ts]: It seems like path should be typed as a string, not any.

File: scripts/test-shim.ts [hunk 3]
```diff
@@ -8,20 +8,22 @@
  * does not support them or has only a partial implementation.
  */
 
+const globalAny:any = global;
```
Comment (captainsafia) [inline scripts/test-shim.ts]: This PR contains changes from the other PR. You might want to remove the commit that brings in these changes from this PR or create a new branch off master for each new PR.

**Review by captainsafia — COMMENTED** [review]
Left some comments inline.

File: packages/fs-observable/__mocks__/fs.ts (hunk 1 above)
Comment (ramantehlan) [reply packages/fs-observable/__mocks__/fs.ts]: without it, it raise the ` Object is of type 'unknown'.` error.

File: packages/fs-observable/__mocks__/fs.ts [hunk 4]
```diff
@@ -1,4 +1,11 @@
-const fs = jest.genMockFromModule("fs");
+type Fs = {
```
Comment (ramantehlan) [inline packages/fs-observable/__mocks__/fs.ts:1]: I know this is not the perfect solution, but I wasn't able to find any other solution.

Comment (captainsafia) [reply packages/fs-observable/__mocks__/fs.ts:1]: This works for now.

**Review by captainsafia — APPROVED** [review]

**Review by captainsafia — APPROVED** [review]
Looks good! Thanks for making this fix!

**Pull request merged**